package.json

```diff
@@ -5,6 +5,7 @@
     "node": ">=12.13.0"
   },
   "scripts": {
+    "audit": "yarn npm audit --recursive --environment production --json > yarn-audit-known-issues",
```

Comment:
`audit` is too generic and might be confused with a broader purpose. Could we name this script something like `update-audit-known-issues`, to make it more express?